Library/Formula/kafka.rb

```diff
@@ -3,18 +3,17 @@
 class Kafka < Formula
   homepage "http://kafka.apache.org"
   head "http://git-wip-us.apache.org/repos/asf/kafka.git"
-  url "http://mirrors.ibiblio.org/apache/kafka/0.8.1.1/kafka-0.8.1.1-src.tgz"
-  mirror "http://mirror.sdunix.com/apache/kafka/0.8.1.1/kafka-0.8.1.1-src.tgz"
-  sha1 "104c15d22da36216a678e6a0c3243c552e47af87"
+  url "http://mirrors.ibiblio.org/apache/kafka/0.8.2.0/kafka-0.8.2.0-src.tgz"
+  mirror "http://mirror.sdunix.com/apache/kafka/0.8.2.0/kafka-0.8.2.0-src.tgz"
+  sha1 "d2c35b60a2f534fb552030dcc7855d13292b2414"
 
+  depends_on "gradle"
   depends_on "zookeeper"
   depends_on :java => "1.7"
 
   def install
-    system "./gradlew", "jar"
-
-    # Use 1 partition by default so individual consumers receive all topic messages
-    inreplace "config/server.properties", "num.partitions=2", "num.partitions=1"
+    system "gradle"
```

Comment:
KAFKA-1490 removed the gradle jar from kafka source and introduced the dependency on gradle. The 0.8.2 readme was updated to include the additional steps - https://github.com/apache/kafka/blob/0.8.2/README.md.